test/test_abi

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# This test helps to keep libnxz's ABI stability. It does the following checks:
+#   1. Make sure that libnxz provides all zlib symbols
+#   2. Guarantee that new symbols are not added/removed by mistake
+
+# Exit on first failure
+set -e
+
+# List of zlib exported symbols currently not implemented by libnxz. These
+# must be ignored in our ABI comparison for now.
+missing_symbols=(
+    crc32_z
+    deflateGetDictionary
+    deflateParams
+    deflatePending
+    deflatePrime
+    deflateTune
+    get_crc_table
+    gzbuffer
+    gzclearerr
+    gzclose
+    gzclose_r
+    gzclose_w
+    gzdirect
+    gzdopen
+    gzeof
+    gzerror
+    gzflush
+    gzfread
+    gzfwrite
+    gzgetc
+    gzgetc_
+    gzgets
+    gzoffset64
+    gzoffset
+    gzopen
+    gzopen64
+    gzprintf
+    gzputc
+    gzputs
+    gzread
+    gzrewind
+    gzseek
+    gzseek64
+    gzsetparams
+    gztell
+    gztell64
+    gzungetc
+    gzvprintf
+    gzwrite
+    inflateBack
+    inflateBackEnd
+    inflateBackInit_
+    inflateCodesUsed
+    inflateGetDictionary
+    inflateMark
+    inflatePrime
+    inflateSync
+    inflateUndermine
+    inflateValidate
+    zError
+    zlibCompileFlags
+)
+
+# These should be either the .so files or their descriptions generated by abidw
+zlib_abi_orig="libz.1.2.11.abi"
+libnxz_abi="libnxz.0.63.0.abi"
```

Comment:
It will be very hard to maintain these tests as zlib and libnxz have newer releases.
Can we use `zlib.abi` and `libnxz.abi` instead?

Reply:
Sure, I'll change that in the next push.